sql/test_data_second_domain.mysql

```diff
@@ -10,13 +10,13 @@ SELECT @addId := id from civicrm_address where street_address = '15 Main St';
 
 INSERT INTO civicrm_email (contact_id, location_type_id, email, is_primary, is_billing, on_hold, hold_date, reset_date)
       VALUES
-      (@contactID, 1, '"Domain Email" <domainemail2@example.org>', 0, 0, 0, NULL, NULL);
+      (@contactID, 1, '"Domain Email" <domainemail2@example.org>', 1, 0, 0, NULL, NULL);
```

Comment:
Is that a valid email?

Reply:
I don't understand the question

Reply:
oh - you mean the part I didn't change looks weird - yeah  it does

Reply:
more like that doesn't seem to be a correct data for that column like that looks more appropriate for the option_value.value rather than civicrm_email

Reply:
yeah - I can change that to just the email  & add in the regen update if this is otherwise ok

Reply:
@seamuslee001 unrelated test fail -  but I've fixed ^^ & grabbed in the regenerate sql commit